Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingFragment.kt

```diff
@@ -33,9 +35,14 @@ class SubmissionTestResultPendingFragment : Fragment(R.layout.fragment_submissio
 
     private var errorDialog: AlertDialog? = null
 
+    private val navArgs by navArgs<SubmissionTestResultPendingFragmentArgs>()
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        pendingViewModel.updateTestType(navArgs.testType)
+        Toast.makeText(context, "Scanned ${navArgs.testType} test", Toast.LENGTH_SHORT).show()
```

Comment:
I think this should not appear to the user ?

Reply:
This should be removed in the end, yes. I just wanted a quick visual feedback to know that the routing works correctly when testing, as I haven't updated the UI with the right strings yet.